web/packages/teleterm/src/ui/components/LinearProgress/LinearProgress.tsx

```diff
@@ -18,35 +18,40 @@
 
 import styled from 'styled-components';
 
-interface LinearProgressProps {
-  transparentBackground?: boolean;
-}
-
-export const LinearProgress = (props: LinearProgressProps) => {
+export const LinearProgress = ({
+  transparentBackground = false,
+  absolute = true,
+  hidden = false,
```

Comment:
Yeah, though idk how good that is. If someone wants to add, say, an enum prop in the future, they'll have to add the whole type anyway instead of just adding one line to an existing type. ;f